fuzz/fuzz_targets/validation.rs

```diff
@@ -0,0 +1,69 @@
+#![no_main]
+use apollo_compiler::ApolloCompiler;
+use apollo_rs_fuzz::generate_valid_operation;
+use libfuzzer_sys::fuzz_target;
+use log::debug;
+use router_bridge;
+// use serde_json::json;
+// use serde_json::Value;
+use std::panic;
+
+fuzz_target!(|data: &[u8]| {
+    let (op, ts) = match generate_valid_operation(data) {
+        Ok(d) => (d.0, d.1),
+        Err(_err) => {
+            return;
+        }
+    };
+
+    let mut compiler = ApolloCompiler::new();
+    compiler.add_executable(&op, "op.graphql");
+    compiler.add_document(&ts, "ts.graphql");
```

Comment:
```suggestion
    compiler.add_type_system(&ts, "ts.graphql");
```
just to match actual usage, the behaviour of `document` is a bit different